lib/smart_proxy_openscap/openscap_api.rb

```diff
@@ -7,49 +7,102 @@
 # FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv3
 # along with this software; if not, see http://www.gnu.org/licenses/gpl.txt
 #
-
 require 'smart_proxy_openscap/openscap_lib'
+require 'pry'
```

Comment:
this should go away, pry is bundled only in development, you can add Gemfile.local to .gitignore and load it from main Gemfile if you want pry to be required on your host